Samples/Blinky/Blinky-samc21xpro.lpi

```diff
@@ -73,33 +73,31 @@
   <CompilerOptions>
     <Version Value="11"/>
     <Target>
-      <Filename Value="Blinky"/>
+      <Filename Value="bin/$(TargetCPU)-$(TargetOS)/samc21/$NameOnly($(ProjFile))"/>
```

Comment:
I guess you did this because you had issues with build. 
There are two problems here:
a) As we heavily use defines it is still unsave to do a simple build, better do a rebuild all from time to time
b) If we stay with this we will need a generic way to define the name of the 'samc21' directory because I also need to define the path to the executable for Debugging (Ozone)

Reply:
No build issues.
I always use a single lpi for multiple targets.
And use "Compile many Modes" to generate all binaries at once.
And use the paths settings to split libraries and binaries for easy comparison.